src/unittests/quench.ts

```diff
@@ -14,10 +14,8 @@ import { shadowrunSR5SpriteDataPrep } from './sr5.SpriteDataPrep.spec';
 import { shadowrunSR5ICDataPrep } from './sr5.ICDataPrep.spec';
 import { shadowrunSR5VehicleDataPrep } from './sr5.VehicleDataPrep.spec';
 import { shadowrunSR5ActiveEffect } from './sr5.ActiveEffect.spec';
-import { shadowrunNetworkDevices } from './sr5.NetworkDevices.spec';
```

Comment:
As both tests currently are 'empty', removal is fine. Though we should still at least reimplement the Network one, as it did contain tests once that aren't up to date.

You could throw a task into the general organisational project for that.